odonto/admin.py

```diff
@@ -71,8 +74,34 @@ def performer_number(self, obj):
         return ''
 
 
+class PatientAdmin(VersionAdmin):
+    list_display = ('__str__', 'patient_detail_link')
+
+    search_fields = [
+        'demographics__first_name',
+        'demographics__surname',
+        'demographics__hospital_number'
+    ]
+
+    def patient_detail_url(self, obj):
+        return reverse('odonto-patient-detail', kwargs={"pk": obj.id})
+
+    def patient_detail_link(self, obj):
+        url = self.patient_detail_url(obj)
+        return format_html(
+            "<a href='{url}'>{url}</a>", url=url
+        )
+
+    def view_on_site(self, obj):
+        return self.patient_detail_url(obj)
+
+    patient_detail_url.short_description = "Patient Detail Url"
```

Comment:
Isn't this on the wrong method?

Reply:
TBH I stole this from https://github.com/openhealthcare/opal/blob/v0.19.0/opal/admin.py#L113

Its not actually needed as it defaults to `Patient Detail Link` which is fine.